python-backend/tests/utils/include/test_user_info.py

```diff
@@ -5,9 +5,10 @@
 from tests.constants import VIEW_ONLY_AUTH_CLAIMS, TOKEN_HEADER
 
 
-def test_get_user_info(test_client):
-    User._test_mode = False
+# FIXME: This test breaks all future http requests, presumably from the patch?
```

Comment:
Is there a ticket made for this fix?  May be a good idea to have a jira ticket to track it.

Reply:
No ticket at this point. I've just opened this as a discussion that should happen. If this PR gets merged without having resolved this issue, I'll create a ticket.